packages/maskbook/src/components/InjectedComponents/SetupGuide.tsx

```diff
@@ -314,19 +396,20 @@ function FindUsername({ username, onConnect, onDone, onClose, onUsernameChange =
                             onKeyDown={onKeyDown}
                             inputProps={{ 'data-testid': 'username_input' }}></TextField>
                         <IconButton
-                            className={findUsernameClasses.button}
+                            className={isMobile ? classes.hide : findUsernameClasses.button}
                             color={username ? 'primary' : 'default'}
                             disabled={!username}>
                             <ArrowRight className={findUsernameClasses.icon} cursor="pinter" onClick={onJump} />
                         </IconButton>
                     </Box>
-
-                    <Typography
-                        className={classes.tip}
-                        variant="body2"
-                        dangerouslySetInnerHTML={{ __html: t('setup_guide_find_username_text') }}></Typography>
                 </form>
             }
+            tip={
+                <Typography
+                    className={classes.tip}
+                    variant="body2"
+                    dangerouslySetInnerHTML={{ __html: t('setup_guide_find_username_text') }}></Typography>
```

Comment:
Can we remove the usage of `dangerouslySetInnerHTML`?

Reply:
@lelenei

Reply:
Leave it as-is. This is not a new added usage to `dangerouslySetInnerHTML`